miio/vacuum.py

```diff
@@ -540,6 +540,21 @@ def get_room_mapping(self):
         """Retrieves a list of segments."""
         return self.send("get_room_mapping")
 
+    @command()
+    def get_recover_maps(self):
+        """Get recover maps."""
```

Comment:
Maybe `get_saved_maps`?

Reply:
Actually its more like recover or backup maps than real saved maps. There is the current map and two backup maps, like this:
![20200102_103607229_iOS](https://user-images.githubusercontent.com/15311791/71663265-c0995b00-2d54-11ea-9fdc-178a939be00e.png)